pkg/roachprod/install/cluster_synced.go

```diff
@@ -2597,24 +2597,33 @@ func (c *SyncedCluster) Get(
 		close(results)
 	}()
 
+	var errOnce sync.Once
+	var finalErr error
+	setErr := func(e error) {
+		if e != nil {
```

Comment:
Nit: There's already a `r.err != nil` check, so this one might not be necessary.

Reply:
Removed.